test/OpenTelemetry.Exporter.Jaeger.Tests/JaegerExporterOptionsTests.cs

```diff
@@ -0,0 +1,80 @@
+// <copyright file="JaegerExporterOptionsTests.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using Xunit;
+
+namespace OpenTelemetry.Exporter.Jaeger.Tests
+{
+    public class JaegerExporterOptionsTests : IDisposable
+    {
+        public JaegerExporterOptionsTests()
+        {
+            this.ClearEnvVars();
+        }
+
+        public void Dispose()
+        {
+            this.ClearEnvVars();
+        }
+
+        [Fact]
+        public void JaegerExporterOptions_Defaults()
+        {
+            var options = new JaegerExporterOptions();
+
+            Assert.Equal("localhost", options.AgentHost);
+            Assert.Equal(6831, options.AgentPort);
+            Assert.Equal(4096, options.MaxPayloadSizeInBytes);
+            Assert.Equal(ExportProcessorType.Batch, options.ExportProcessorType);
+        }
+
+        [Fact]
+        public void JaegerExporterOptions_EnvironmentVariableOverride()
+        {
+            Environment.SetEnvironmentVariable(JaegerExporterOptions.OTelAgentHostEnvVarKey, "jeager-host");
+            Environment.SetEnvironmentVariable(JaegerExporterOptions.OTelAgentPortEnvVarKey, "123");
+
+            var options = new JaegerExporterOptions();
+
+            Assert.Equal("jeager-host", options.AgentHost);
+            Assert.Equal(123, options.AgentPort);
+        }
+
+        [Fact]
+        public void JaegerExporterOptions_InvalidPortEnvironmentVariableOverride()
```

Comment:
could you add a test to validate that in-code takes priority over the environment ones? https://github.com/open-telemetry/opentelemetry-dotnet/pull/1924/files#r601893949

Reply:
Sure, thanks for the hyperlink to the PR. I have not seen it 😄

Reply:
Will do

Reply:
Addressed 72e9c4e7